recipes/scala/README.md

```diff
@@ -13,330 +13,112 @@ Sample Scala Recipe using the retail data.
-        }
-        
-### Upload Dataset from UI             
-
-Upload the json from here:
-`https://github.com/adobe/experience-platform-dsw-reference/blob/master/datasets/retail/XDM0.9.9
-.9/DSWRetailSalesForXDM0.9.9.9.json`
-
-
-Create a dataset with "SchemaForRetailData" schema and the json file as the source. Replace the tenantId in the
-json datafile and use this as the data source.
-Please look at the `Video to create dataset and upload file` to platform UI.
-
-### Video to create dataset and upload file
-
-[![Watch the video](../../docs/images/HomePage.png)](https://youtu.be/v1QIlCe5dgw)
-
-### Upload output schema 
-
-Upload the output schema (schema should have "prediction:Number, store:Integer, date:String"). 
-Please refer to the curl commands above to Create Class, Create Mixin, Create Schema for output data with naming them
-appropriately. In this example, the output class is named "ClassForRetailOutput", output mixin is named 
-"MixinForRetailOuputData" and output schema is "SchemaForRetailOutputData"
-The mixin should have the following properties:
-
-        "properties": {
-               "date": {
-                  "title": "date",
-                  "type": "string",
-                  "description": "date"
-            	},
-            	"store": {
-                  "title": "store",
-                  "type": "integer",
-                  "description": "store"
-            	},
-            	"prediction": {
-                  "title": "prediction",
-                  "type": "number",
-                  "description": "prediction"
-            	}
-            	
-        }
-        
-Create an empty dataset with this schema. 
-Please look at the video `Video to create dataset and upload file`. 
-(Do not have to upload any json for this because we want an empty dataset)
-Get this datasetId and plug it in pipelineservice.json. For eg: it is referenced as scoredDatasetId in the sample json. All the prerequisites are complete and you can now proceed to running some training and scoring jobs. 
-
+To run the recipe in any org, we would need the schema of the dataset,the input dataset, 
+output schema and empty output dataset uploaded to the platform UI. For setting this up, use the bootstrap script 
+within `/acp-data-services-dsw-reference/bootstrap`
+Get the tenant id from the bootstrap script and replace the value in the `pipelineservice.json` 
+If you got the engine artifact from the bootstrap script, jump to the section `Video for Training, Scoring and 
+Saving data` 
 
 # Steps to run training and scoring jobs
-
 git clone this repository from `https://github.com/adobe/experience-platform-dsw-reference`
 
-
 cd to recipes/scala and run `build.sh` 
 
 ```
 sh ./build.sh
 ```
 
 This generates a fat jar and is saved in the target directory of the project. 
-Use this jar and go to platform UI and run training and scoring. 
+Use this jar and go to platform UI `https://platform.adobe.io` and run training and scoring. 
```

Comment:
https://platform.adobe.io renders a 404.  Should this be, https://platform.adobe.com/ ?